utils/syntax_highlighting.jsx

```diff
@@ -127,12 +127,40 @@ hlJS.registerLanguage('yaml', hljsYaml);
 
 const HighlightedLanguages = Constants.HighlightedLanguages;
 
+// This function add line numbers to code
+function formatHighLight(code) {
+    if (code) {
+        const width = code.split('\n').length > 1000 ? '36px' : '30px';
```

Comment:
I wonder if we shouldn't have these values on css, and use different classes for it